daemon/container_unix.go

```diff
@@ -640,7 +640,7 @@ func (container *Container) updateNetworkSettings(n libnetwork.Network, ep libne
 		return err
 	}
 
-	if container.hostConfig.NetworkMode == runconfig.NetworkMode("bridge") {
+	if container.hostConfig.NetworkMode == runconfig.NetworkMode("default") {
```

Comment:
`default` can be any network plugin that may or may not use bridge interface.
Hence depending on it isn't appropriate.

`n.Type()` will return the driver used to create the network. This can be combined with `NetworkMode.IsBridge()` to find if we are using bridge driver. But, with the upcoming networking features, user will be able to create multiple drivers using bridge driver. And each network will have its dedicated bridge. Hence, we need a more generic way to address this.